apps/app/src/test/scala/org/lfdecentralizedtrust/splice/integration/tests/SvFrontendIntegrationTest.scala

```diff
@@ -564,6 +730,24 @@ class SvFrontendIntegrationTest
       }
     }
 
+    "NEW UI: Offboard" in { implicit env =>
+      val sv3PartyId = sv3Backend.getDsoInfo().svParty.toProtoPrimitive
+
+      createProposal("SRARC_OffboardSv") { implicit webDriver =>
```

Comment:
Once createProposal() works, it should be pretty easy to complete the tests for the other proposals since they all use createProposal.